packages/cli/src/commands/db.js

```diff
@@ -1,15 +1,38 @@
-export const command = 'db <command>'
-export const aliases = ['database']
-export const description = 'Database tools'
 import terminalLink from 'terminal-link'
 
-export const builder = (yargs) =>
-  yargs
-    .commandDir('./dbCommands')
-    .demandCommand()
-    .epilogue(
-      `Also see the ${terminalLink(
-        'Redwood CLI Reference',
-        'https://redwoodjs.com/reference/command-line-interface#db'
-      )}`
-    )
+import c from 'src/lib/colors'
+
+export const command = 'db [...commands]'
+
+export const builder = () => {
+  const argv = process.argv.slice(2)
+
+  console.log(c.warning('Deprecation notice:'))
+  console.log(
+    `The ${c.bold(
+      'rw ' + argv.join(' ')
+    )} command is now deprecated, please use ${c.bold(
+      `rw prisma ` + (argv[1] || '\b')
+    )} instead.`
+  )
+
+  switch (argv[1]) {
+    case 'up':
+    case 'down':
+    case 'save':
```

Comment:
We can probably be way more hand-holdy here; it would be nice to write this out and decide what works best before trying to implement it.

Reply:
take a look at my additions!